spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/ModuleDefinitionBuilder.java

```diff
@@ -98,15 +98,15 @@ else if (argument.getName().equalsIgnoreCase("outputType")) {
 			}
 			builders.add(builder);
 		}
-		SourceChannelNode sourceChannel = streamNode.getSourceChannelNode();
+		SourceDestinationNode sourceChannel = streamNode.getSourceDestinationNode();
```

Comment:
should change the variable names too... basically a case-insensitive search for "channel" in this file should find nothing